### ISSUE TITLE
Add notice to Moderator Report UI if post has CW but user disables CWs

### ISSUE THREAD

**Issue by ThisIsMissEm** [mastodon/mastodon#29291]
### Pitch

There's a semi-frequent case, particularly for NSFW creators, where they end up getting suspended from other servers despite CW'ing their content appropriately.

This seems to be caused or inflamed by the "disable CWs" option.

In order to improve this situation we should notify moderators that the user who reported the content has CWs disabled, and that might be why they saw content they didn't like. Along with encouraging user-blocking if need be.

### Motivation

This aims to give moderators more information to take appropriate moderation decisions.

**Comment by ClearlyClaire** [comment]
What do you mean by “disable CWs option”?

**Comment by trwnh** [comment]
i assume the "always expand posts even if they have a CW" option

**Comment by ThisIsMissEm** [comment]
Yeah, from time to time I've come across people saying "I've received this report about content that shouldn't be on mastodon" (nudity, lewd, etc), and it turns out that the author of the report has explicitly disabled the content warnings by setting "always expand posts marked with content warnings" option; so the report isn't necessary because the author did actually mark it correctly, the end user just decide that they didn't like content warnings.

**Comment by ClearlyClaire** [comment]
Alright, I understand the situation better. That being said, it should already visible through the report whether something has a CW or is marked sensitive, so you should already be able to tell?

I suppose it might also be worth reminding the user of those settings in the report modal.

**Comment by trwnh** [comment]
> it should already visible through the report whether something has a CW or is marked sensitive, so you should already be able to tell?

as a mod, you can tell that the post has a CW, but you can't tell that the reporter has expanded the CW by default. the issue seems to be asking for the latter: indicate to the moderator that the person who has reported a post has that setting enabled.

**Comment by ThisIsMissEm** [comment]
@trwnh yeah, that's correct — basically you get moderators being confused "why is this person reporting correctly CW'd content?" and many times it's because they've not understood the settings and set it to "always expand posts with content warnings", so this surfaces that information to moderators as to help clarify why a CW'd post might have been reported.

**Comment by ThisIsMissEm** [comment]
Have opened #31475 to address this, however, to the remote server (the source of the report) the report UI wouldn't show this warning, since it doesn't have access to the reporting users' settings.
